plenum/server/node.py

```diff
@@ -406,6 +395,97 @@ def __init__(self,
         # We need future_primaries to calculate applied primaries correctly
         self.future_primaries_handler = FuturePrimariesBatchHandler(self.db_manager, self)
 
+    def init_req_managers(self):
+        self.write_manager = WriteRequestManager(self.db_manager)
+        self.read_manager = ReadRequestManager()
+        self.action_manager = ActionRequestManager()
+
+    def init_storages(self, storage=None):
```

Comment:
What about Plugins (plugins-specific storages, ledgers, req handlers)? How are we going to init and register them?

Reply:
Maybe it will be better to create separate classes (Builders/Factories) for every ledger?

Reply:
ledgers and states can be registered by calling node.db_manager.register_new_database method, req/batch handlers using node.write_manager and node.read_manager for write and read requests.